test/docx/body-reader.tests.js

```diff
@@ -861,6 +873,23 @@ function createLinkedBlip(relationshipId) {
     return new XmlElement("a:blip", {"r:link": relationshipId});
 }
 
+function createHyperlink(link) {
```

Comment:
Given that there are two ways of creating hyperlinks, the name of this should probably be more explicit so that it describes what sort of hyperlink it's creating.